src/main/java/de/tum/in/www1/artemis/service/ProgrammingExerciseExportService.java

```diff
@@ -87,8 +87,9 @@ public File exportStudentRepositories(long programmingExerciseId, @NotNull List<
             log.info("Request to export all student repositories of programming exercise " + programmingExerciseId + " with title '" + programmingExercise.getTitle() + "'");
         }
         else {
+            // TODO: Is this the expected behavior for teams? Or should all team members be listed instead?
```

Comment:
The TODO placement might have been a little misleading; it refers to the whole method and not just the log statement. But I think you are right that only @krusche has enough insight here to know how `exportStudentRepositories` should behave for teams. The behavior right now is that it treats teams as it would treat students (and uses the team shortName for the student login).

Reply:
should be ok